content/blog/2025/12/2025-12-12-twim.md

```diff
@@ -0,0 +1,196 @@
++++
+date = "2025-12-12T17:19:38+00:00"
+title = "This Week in Matrix 2025-12-12"
+path = "/blog/2025/12/12/this-week-in-matrix-2025-12-12"
+
+[taxonomies]
+author = ["Andrew Morgan (anoa) {he/him}"]
```

Comment:
[please match existing authors](https://github.com/matrix-org/matrix.org/blob/main/CONTENT.md#publishing-to-the-blog): https://matrix.org/author/andrew-morgan/